src/pools/composableStable/composableStablePool.ts

```diff
@@ -112,7 +115,8 @@ export class ComposableStablePool extends PhantomStablePool {
         amount: OldBigNumber
     ): OldBigNumber {
         try {
-            // This code assumes that decimalsIn and decimalsOut is 18
+            // This code needs decimalsIn and decimalsOut as 18
```

Comment:
I find this comment confusing - decimalsIn/Out will actually be related to the token decimals which aren't always going to be 18.
The values used should use 1e18 fixed point - as mentioned in comment below.